restio/model.py

```diff
@@ -55,11 +59,15 @@ def __new__(cls, name: str, bases: Tuple[Type, ...], dct: Dict[str, Any]):
         meta = ModelMeta()
         dct["_meta"] = meta
 
-        def _update_meta(_meta: ModelMeta, extend: bool):
+        def _update_meta(
+            _meta: ModelMeta, extend: bool, not_inherited: Tuple[str, ...] = tuple()
+        ):
             if not _meta:
```

Comment:
Not that this is necessarily in the PR, but if this check needs to be here, does that mean the `_meta` parameter should type should be `Optional`?

Reply:
You are absolutely right, let me use this PR to also fix that